restorer.go

```diff
@@ -44,8 +45,20 @@ func (res *Restorer) restoreTo(dst string, dir string, treeID backend.ID) error
 	}
 
 	for _, node := range tree.Nodes {
-		if err := res.restoreNodeTo(node, dir, dst); err != nil {
-			return err
+		selectedForRestore := true
+		if res.SelectForRestore != nil {
```

Comment:
maybe it would be nicer to initialize `res.SelectForRestore` to `func(...) { return true }`, then you wouldn't have to do this kind of check... dunno if that's necessarily better. just an idea.